modules.d/45url-lib/module-setup.sh

```diff
@@ -32,9 +32,9 @@ install() {
         for _lib in "$dracutsysrootdir$_dir"/libcurl.so.*; do
             [[ -e $_lib ]] || continue
             if ! [[ $_nssckbi ]]; then
-                read -r -d '' _nssckbi < <(grep -F --binary-files=text -z libnssckbi "$_lib")
+                read -r -d '' _nssckbi <<< "(grep -F --binary-files=text -z libnssckbi $_lib)"
             fi
-            read -r -d '' _crt < <(grep -F --binary-files=text -z .crt "$_lib")
+            read -r -d '' _crt <<< "(grep -F --binary-files=text -z .crt $_lib)"
```

Comment:
missing `$`

Reply:
Are you sure? I mean this works just fine without it and adding it triggers warning: command substitution: ignored null byte in input due to grep -z and at that point we just down a rabbit hole that my brain cant cope with at the moment :)

Reply:
Yes, without `$` the grep command is never executed:
```bash
$ read -r -d '' _crt <<< "(grep -F --binary-files=text -z .crt $_lib)"; declare -p _crt
declare -- _crt="(grep -F --binary-files=text -z .crt )"
```

And if the `grep` command is executed, the string will contain null bytes. This is why we don't convert it to a string in the first place and use `read` with `< <()`